external/fv3fit/fv3fit/emulation/losses.py

```diff
@@ -60,7 +65,8 @@ def build(self, output_samples: Mapping[str, tf.Tensor],) -> Callable:
         """
         loss_funcs = {}
         for out_varname, sample in output_samples.items():
-            loss_funcs[out_varname] = NormalizedMSE(self.normalization, sample)
+            norm_layer = self.normalization.build(sample)
+            loss_funcs[out_varname] = NormalizedMSE(norm_layer)
```

Comment:
this shows the benefit of passing an instantiated object to a class constructor rather than constructing the normalization inside. Makes it possible to pass any callable to NormalizedMSE. Probably would be even better to just pass the scales.